src/reset-password/ResetPasswordPage.jsx

```diff
@@ -120,7 +121,9 @@ const ResetPasswordPage = (props) => {
   } else {
     return (
       <>
-
+        <Helmet>
```

Comment:
I guess this needs to be added on invalid token and password reset success cases as well.